src/GADM.jl

```diff
@@ -175,4 +175,21 @@ function get(country, levels...)
     throw(ArgumentError("feature not found"))
 end
 
+"""
+    coordinates(country, levels...)
+Returns a deep array of Float64 coordinates for the requested region.\n
+Input: ISO 3166 Alpha 3 Country Code, and further full official names of subdivisions\n
+
+## Examples  
+  
+`coordinates("IND")` # Returns array of coordinates of India  
+`coordinates("IND", "Uttar Pradesh")` # Returns array of coordinates of the state Uttar Pradesh  
+`coordinates("IND", "Uttar Pradesh", "Lucknow")` # Returns array of coordinates of district Lucknow  
```

Comment:
I think the example needs to just print the beginning of an array returned by the function. The syntax is the same as `GADM.polygon` and so you could point the user to the other docstring.